modules/encrypt/config/encrypt.php

```diff
@@ -3,6 +3,7 @@
 return array(
 
 	'default' => array(
+		'type'       => 'mcrypt',
```

Comment:
I think this van be changed to the new default, existing installations have an existing config.